run/filesystem/test/e2e_test_setup.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+- id: 'Build and deploy to Cloud Run'
+  name: 'gcr.io/cloud-builders/gcloud:latest'
+  entrypoint: /bin/bash
+  args:
+  - '-c'
+  - |
+    gcloud run deploy $_RUN_SERVICE --source . \
+    --region $_REGION \
+    --vpc-connector $_CONNECTOR_NAME \
+    --execution-environment gen2 \
+    --allow-unauthenticated \
```

Comment:
To be safe, let's deploy using `--no-allow-unauthenticated` and adding a token to our test request

Reply:
updated!